<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackLearnDetect</class>
 <widget class="QDialog" name="TrackLearnDetect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Track Learn Detect</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_general">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QScrollArea" name="scrollArea_general">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_general">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>621</width>
            <height>379</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QGroupBox" name="loadVideo">
             <property name="title">
              <string>Load Video</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_modelPath_2">
                <property name="text">
                 <string>Video</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="videoPath">
                <property name="text">
                 <string>./video.mpg</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_loadVideo">
                <property name="text">
                 <string>Browse...</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="saveResults">
             <property name="title">
              <string>Save Results</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_outputDir_2">
                <property name="text">
                 <string>Directory</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="resultsDirectory">
                <property name="text">
                 <string>./</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_saveResults">
                <property name="text">
                 <string>Browse...</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="loadIni">
             <property name="title">
              <string>Load Initialization</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_modelPath">
                <property name="text">
                 <string>File</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="iniPath"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_loadIni">
                <property name="text">
                 <string>Browse...</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="saveIni">
             <property name="title">
              <string>Save Initialization</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_outputDir">
                <property name="text">
                 <string>Directory</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="iniDirectory">
                <property name="text">
                 <string>./</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_saveIni">
                <property name="text">
                 <string>Browse...</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_detector">
      <attribute name="title">
       <string>Detector</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QScrollArea" name="scrollArea_general_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_general_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>606</width>
            <height>1638</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="Lwin_size_width">
             <property name="text">
              <string>win_size_width</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="win_size_width">
             <property name="text">
              <string>15</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lwin_size_height">
             <property name="text">
              <string>win_size_height</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="win_size_height">
             <property name="text">
              <string>15</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="LLevel">
             <property name="text">
              <string>level</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="level">
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lmin_eigen">
             <property name="text">
              <string>min_eigen</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="min_eigen">
             <property name="text">
              <string>0.025</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lmin_forward_backward_error">
             <property name="text">
              <string>min_forward_backward_error</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="min_forward_backward_error">
             <property name="text">
              <string>100</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Linterval">
             <property name="text">
              <string>interval</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="interval">
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lshift">
             <property name="text">
              <string>shift</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="shift">
             <property name="text">
              <string>0.5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lmin_win">
             <property name="text">
              <string>min_win</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="min_win">
             <property name="text">
              <string>20</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="include_overlap">
             <property name="text">
              <string>include_overlap</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="include_overlap_2">
             <property name="text">
              <string>0.4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lexclude_overlap">
             <property name="text">
              <string>exclude_overlap</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="exclude_overlap">
             <property name="text">
              <string>0.7</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lstructs">
             <property name="text">
              <string>structs</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="structs">
             <property name="text">
              <string>40</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lfeatures">
             <property name="text">
              <string>features</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="features">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lvalidate_set">
             <property name="text">
              <string>validate_set</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="validate_set">
             <property name="text">
              <string>0.5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lnnc_same">
             <property name="text">
              <string>nnc_same</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nnc_same">
             <property name="text">
              <string>0.60</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lnnc_thres">
             <property name="text">
              <string>nnc_thres</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nnc_thres">
             <property name="text">
              <string>0.65</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lnnc_verify">
             <property name="text">
              <string>nnc_verify</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nnc_verify">
             <property name="text">
              <string>0.7</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lnnc_beyond">
             <property name="text">
              <string>nnc_beyond</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nnc_beyond">
             <property name="text">
              <string>0.8</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lbad_patches">
             <property name="text">
              <string>bad_patches</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="bad_patches">
             <property name="text">
              <string>100</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lnew_deform">
             <property name="text">
              <string>new_deform</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="new_deform">
             <property name="text">
              <string>20</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Ltrack_deform">
             <property name="text">
              <string>track_deform</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="track_deform">
             <property name="text">
              <string>40</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lnew_deform_angle">
             <property name="text">
              <string>new_deform_angle</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="new_deform_angle">
             <property name="text">
              <string>20</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Ltrack_deform_angle">
             <property name="text">
              <string>track_deform_angle</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="track_deform_angle">
             <property name="text">
              <string>50</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lnew_deform_scale">
             <property name="text">
              <string>new_deform_scale</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="new_deform_scale">
             <property name="text">
              <string>0.02</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Ltrack_deform_scale">
             <property name="text">
              <string>track_deform_scale</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="track_deform_scale">
             <property name="text">
              <string>0.02</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lnew_deform_shift">
             <property name="text">
              <string>new_deform_shift</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="new_deform_shift">
             <property name="text">
              <string>0.02</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Ltrack_deform_shift">
             <property name="text">
              <string>track_deform_shift</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="track_deform_shift">
             <property name="text">
              <string>0.02</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Ltop_n">
             <property name="text">
              <string>top_n</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="top_n">
             <property name="text">
              <string>100</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lrotation">
             <property name="text">
              <string>rotation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="rotation">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lnnc_collect">
             <property name="text">
              <string>nnc_collect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nnc_collect">
             <property name="text">
              <string>0.5</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButton_loadIni</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>iniPath</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
